nservicebus/tools/migrate-to-native-delivery.md

```diff
@@ -266,3 +308,23 @@ To list the status of timeouts for an a previous/in-progress run take the `Migra
 `SELECT * FROM TimeoutData_migration_{MigrationRunId}`
 
 This will show all the timeouts and to which batch they belong and also that status of that batch, `0=Pending`, `1=Staged` and `2=Completed`.
+
+### Azure Storage persistence
+
+To list the history and status of migrations open up the `timeoutsmigrationtoolstate` Azure Table inside the Storage Explorer.
+
+The migrated timeouts as well as their status can be inspected in the `timeoutsmigration` Azure Table. This will show all the timeouts and to which batch they belong and also that status of that batch, `0=Pending`, `1=Staged` and `2=Completed`.
```

Comment:
```suggestion
The migrated timeouts as well as their status can be inspected in the `timeoutsmigration` Azure Table. This will show all the timeouts, to which batch they belong, and its status: `0=Pending`, `1=Staged` and `2=Completed`.
```

Reply:
I think we should apply then the same slightly higher above. I like the suggestion